roles/setup_pgd/tasks/pg_hba_config.yml

```diff
@@ -57,6 +57,64 @@
   run_once: true
   no_log: "{{ disable_logging }}"
 
+- name: Prepare hba non ssl list for user - pg_owner
+  ansible.builtin.set_fact:
+    pg_allow_ip_addresses: >-
+      {{ pg_allow_ip_addresses | default([]) + [
+        {
+          "contype": "host",
+          "users": pg_owner,
+          "source": node.private_ip + "/32",
+          "databases": "postgres"
+          },
+        {
+          "contype": "host",
+          "users": pg_owner,
+          "source": node.private_ip + "/32",
+          "databases": "postgres"
+          },
+          {
+           "contype": "host",
+           "users": pg_owner,
+           "source": node.private_ip + "/32",
+           "databases": "postgres"
+           }
+        ] }}
+  when: not pg_ssl
+  loop: "{{ pgd_cluster_nodes }}"
+  loop_control:
+    loop_var: node
+  run_once: true
+  no_log: "{{ disable_logging }}"
+
+- name: Prepare hba ssl list for user - pg_owner
+  ansible.builtin.set_fact:
+    pg_allow_ip_addresses: >-
+      {{ pg_allow_ip_addresses | default([]) + [
+        {
+          "users": pg_owner,
+          "source": node.private_ip + "/32",
+          "databases": "postgres"
+          },
+          {
+          "contype": "host",
```

Comment:
this should either be absent or read `"contype": "hostssl"`

Reply:
I think since Doug has proofed that it worked, this is more of a question of whether we want to rely on the default values we set in manage_dbserver.manage_hba_conf, vs explicitly setting this knowing it is the same as the default values.

So I think it's ok to leave as is unless we want to be more explicit everywehre.